magento/glossary
# Project directory cleanup

## Purpose of this PR

Cleans up the project directory:

* Deletes the `json-to-markdown-converter` script since it's no longer needed
* Deletes the json files in the `terms` directory and replace them with the markdown files in the `_converted` directory
* Deletes the `_converted` directory

Fixes #14 

## Validation steps

1. Delete `public` and `.cache` folders
2. Run `yarn install` if project dependencies not yet installed
3. Run `yarn build`
4. Verify the site builds successfully with no errors
5. (Optional) Run `yarn develop` and verify site looks the same
